frontend/src/App.css

```
@@ -493,6 +494,386 @@ body {
+}
+
+.chat-bubble-user .chat-bubble {
+  background: var(--color-primary);
+  color: #fff;
+  border-color: var(--color-primary);
+}
+
+.chat-bubble-content {
+  font-size: 14px;
+  line-height: 1.6;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)